shared-bindings/bitmaptools/__init__.c

```diff
@@ -522,6 +522,69 @@ STATIC mp_obj_t bitmaptools_obj_draw_line(size_t n_args, const mp_obj_t *pos_arg
 MP_DEFINE_CONST_FUN_OBJ_KW(bitmaptools_draw_line_obj, 0, bitmaptools_obj_draw_line);
 // requires all 6 arguments
 
+//| def draw_polygon(
+//|     dest_bitmap: displayio.Bitmap,
+//|     xs: ReadableBuffer,
+//|     ys: ReadableBuffer,
```

Comment:
`xs` and `ys` are typed as ReadableBuffer here, but the same arguments are marked as `int` in the list of params a few lines below here. 

I was able to test this successfully on a PicoW with EYESPI connected ST7789 display using code like this:
```
xs = bytes([4, 101, 101, 19])
ys = bytes([4, 19,  121, 101])
bitmaptools.draw_polygon(color_bitmap, xs, ys, 3)
```

I think it would be best if the arguments `xs` and `ys` could be a list or tuple instead of needing to be specifically the bytes buffer thing.

The vectorio module has a polygon class that accepts a list of tuples for the points that make the polygon. Maybe we can take inspiration from that and try to match this one to the same types where possible. 

Even if we do stick with the bytes buffer type things I think we'll need to update the :params list below to change `int` to the other type.

Reply:
`int` in :params list is a mistake, I forgot to change it there.

I think it would be best to allow both `List[int]` and  `ReadableBuffer` as argument, as first is easy to use, and second allows for optimisations such as using not only `bytes` but also `array` of shorts for less memory usage, or even `memoryview` over such to avoid data copying. I'll try to make it accept both versions (-: